doc/architecture/domains/authorization/features/accept-terms.md

```diff
@@ -0,0 +1,281 @@
+# Terms Acceptance Documentation
+
+## Overview
+
+The Terms Acceptance API allows organizations to accept the latest terms and conditions.
+When an organization accepts the terms, the Authorization database is updated, and an integration event is published,
+to RabbitMQ, using MassTransit's transactional outbox pattern.
+
+## Accept terms flow:
+
+```mermaid
+sequenceDiagram
+    actor User
+    participant API as Terms Endpoint
+    participant Handler as Command Handler
+    participant DB as Database
+    participant RabbitMQ as RabbitMQ Broker
+
+    User->>API: POST /api/authorization/terms/accept
+    activate API
+    API->>API: Validate JWT and extract claims
+    API->>Handler: Send AcceptTermsCommand
+    activate Handler
+
+    rect rgb(50, 50, 50)
+        Note over Handler, DB: Database Interactions
+        Handler->>DB: Begin transaction
+        activate DB
+        Handler->>DB: Get or create organization
+        DB-->>Handler: Return organization
+        Handler->>DB: Fetch latest terms
+        DB-->>Handler: Return latest terms
+        Handler->>DB: Update organization's terms acceptance
+        DB-->>Handler: Confirm update
+        Handler->>DB: Commit transaction
+        DB-->>Handler: Confirm commit
+        deactivate DB
+    end
+
+    rect rgb(50, 50, 50)
+        note over Handler, RabbitMQ: Publish integration event
+        Handler->>RabbitMQ: Send OrgAcceptedTerms message
+    end
+
+    Handler-->>API: Return true
+    deactivate Handler
+
+    API-->>User: 200 OK (Terms accepted successfully)
+    deactivate API
+
+    alt Terms Acceptance Fails
+        activate Handler
+        Handler-->>API: Return false
+        deactivate Handler
+        activate API
+        API-->>User: 400 Bad Request (Failed to accept terms)
+        deactivate API
+    end
+```
+
+## Endpoint
+
+`
+POST /api/authorization/terms/accept
+`
+## Request
+
+The endpoint expects an empty body. All necessary information is derived from the users' claims in their JWT token.
+
+### Authorization
+
+The endpoint requires authorization with the `B2CCvrClaim policy`.
+
+This policy ensures that the user attempting to accept the terms is indeed affiliated with the organization,
+they are accepting the terms on behalf of.
+
+The policy is implemented as a custom authorization policy,
+in the [EnergyOrigin.TokenValidation NuGet package](../../../../../libraries/dotnet/EnergyOrigin.TokenValidation/README.md).
+
+## Response
+
+### Success (200 OK)
+
+```json
+{
+  "status": true,
+  "message": "Terms accepted successfully."
+}
+```
+
+### Failure (400 Bad Request)
+```json
+{
+  "status": false,
+  "message": "Failed to accept terms."
+}
+```
+
+### Failure (403 Forbidden)
+
+```json
+{}
+```
```

Comment:
Why is this needed? We have a swagger :)

Reply:
Removed: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/0e3afd76de61829946609d649dee4927457e4e2a

Reply:
Deleted some more: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/a95f20d81b0d6ddd9af3b7939c59ec9946dc6a73